### ISSUE TITLE
GU0009 should not warn here

### ISSUE THREAD

**Issue by JohanLarsson** [JohanLarsson/Gu.Analyzers#103]
```c#
Assert.AreEqual(true, success);
```

Same for xunit and mstest, above is for nunit.
